plugins/versionlock.py

```diff
@@ -57,6 +58,9 @@ def config(self):
                        and cp.get('main', 'locklist'))
 
     def sack(self):
+        if not self.cli.demands.resolving:
```

Comment:
@j-mracek , right, the hackish implementation of versionlock by applying excludes should be hidden from user because by common sense he don't expect packages to be excluded at all, just have them locked in concrete version -> changed to debug, modified log message and documented.